Library/Formula/smpeg.rb

```diff
@@ -0,0 +1,46 @@
+class Smpeg < Formula
+  desc "SDL MPEG Player Library"
+  homepage "http://icculus.org/smpeg/"
+  url "svn://svn.icculus.org/smpeg/tags/release_0_4_5/"
```

Comment:
Let's add a `revision` to these `url`s as well - It just offers us a little security in terms of making sure the release doesn't quietly move around. In this case:

```ruby
url "svn://svn.icculus.org/smpeg/tags/release_0_4_5/", :revision => "412"
```